ssv/value_check.go

```diff
@@ -39,6 +39,7 @@ func AttesterValueCheckF(
 	network types.BeaconNetwork,
 	validatorPK types.ValidatorPK,
 	validatorIndex phase0.ValidatorIndex,
+	sharePublicKey []byte,
```

Comment:
that should probably be the validator pk no?

Reply:
We can use validator PK.
in this scenario we have to
 -  adjust AddShare signature to pass the validator PK
https://github.com/bloxapp/ssv-spec/blob/17ac6f1047cea3697b60e677d8c5485d9f9ad627/types/signer.go#L59
and store share private key by validator key and not share public key
 - adjust SignBeaconObject sig to pass validator PK
 https://github.com/bloxapp/ssv-spec/blob/17ac6f1047cea3697b60e677d8c5485d9f9ad627/types/signer.go#L34
 in order to check slashing protection before signing

Reply:
Probably best left to when we redo the remote signer